src/configuration/app/timezone.md

```diff
@@ -0,0 +1,23 @@
+# System timezone
+
+All Platform.sh containers default to running in UTC time.  Applications and application runtimes may elect to use a different timezone but the container itself runs in UTC.  That is usually fine and a correct configuration but occasionally there is cause to run the server in a specific timezone.
+
+## Setting the system timezone
+
+The system timezone for any service or app container may be set with the `timezone` property, whose value is one of the [tz database region codes](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) such as `Europe/Paris` or `America/New_York`.
```

Comment:
I presume it affects logs, too.  Logs and cron are about the only things to be affected.  We should verify.